### ISSUE TITLE
Citric acid cycle (TCA cycle) - imported from: Reactome  (R-HSA-71403)

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#208]
- [ ] Make mouse model using model copy.

**Comment by rwst** [comment]
Note that UniProt SDHA links to https://www.rhea-db.org/rhea/40523 a quinone + succinate = a quinol + fumarate. IE it is seen as one reaction, not two on different parts of the complex. I agree with moving the three pyruvate reactions into the pyruvate pathway. I'll have to read about the reverse reactions.

**Comment by rwst** [comment]
What is your opinion on the cytosolic enzymes
- https://www.uniprot.org/uniprotkb/O75874/entry
- https://www.uniprot.org/uniprotkb/P21399/entry
- https://www.uniprot.org/uniprotkb/P40925/entry
- https://www.uniprot.org/uniprotkb/P07954/entry (cytosolic isoform)?

They are annotated to GO:0006099 (TCA cycle). also to the UniProt TCA keyword, and the review [1] places them (and some more) in the TCA cycle by connecting cytosol and mitochondrial matrix via transporters maintaining a metabolite equilibrium. I think I will place them in a separate sub-pathway, same with the transporters. But I really would appreciate your and @deustp01 's opinion here.

[1] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205302/

**Comment by ukemi** [comment]
@deustp01 and I will discuss this a bit more, but I am leaning towards agreeing. Pathway boundaries are always difficult. If you were to create the sub-pathway, what would you call it?

**Comment by rwst** [comment]
Cytosolic extension of citric acid cycle?

**Comment by deustp01** [comment]
The authors of the review Ralf mentions look like they are arguing that the name, "TCA cycle" is fine but that its definition should be broadened to include activities of cytosolic isoforms of mitochondrial TCA cycle enzymes

> The TCA cycle is a central pathway in the metabolism of sugars, lipids and amino acids [[1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205302/#R1)]. The original investigations of the TCA cycle took place in the context of oxidative metabolism, and it is often presented in a simplistic perspective of a cyclic pathway constantly oxidizing the acetyl moiety of acetyl-CoA to CO2, generating NADH and FADH2, which feed electrons to the respiratory chain ([Figure 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205302/figure/F1/) and [Box 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205302/#BX1)). The work that defined the TCA cycle was performed using whole tissue homogenates. When differential centrifugation techniques allowed the preparation of pure mitochondria, it became evident that although mitochondria were necessary and sufficient to perform the cycle, isoforms of some of the enzymes were also found in the cytoplasm not associated with mitochondria ([Table 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205302/table/T1/)).

words

**Comment by ukemi** [comment]
So that would mean they wouldn't be a subpathway in Reactome, but just an addition of reactions to R-HSA-71403?

**Comment by deustp01** [comment]
> just an addition of reactions to R-HSA-71403?

Yes, by default, and an interesting part of the re-annotation will be to see if we can evidence to make them all fit neatly, per the long comment above.

**Comment by rwst** [comment]
Right now coming to the SDH reaction which, I believe, should not be separated into succinate/fumarate and Q10/Q10H2 parts because in catalyzed reactions electron movements on fixed (covalently bounded) parts of the enzyme are never annotated. At the moment Reactome has "SDH complex (ox.)" (which has FAD bound) and "SDH complex (red.)" (FADH2) but the timeframe where FADH2 exists must be extremely short, as the electrons are immediately passed to the three Fe-S clusters and to Q10. So, the electron movement is internal like in many other catalysts, and this is one reaction Q10 + succinate => Q10H2 + fumarate, which is a specialization of  https://www.rhea-db.org/rhea/40523 a quinone + succinate = a quinol + fumarate. Does it make sense?

**Comment by rwst** [comment]
As to the reverse reactions, there is no UniProt annotation to "human GO:0019643". However, there are recent findings in [1] which would rather argue for a separate pathway. They also give references to reverse reactions in cancer metabolism. And they have use for some cytosolic enzymes, too. So I think I'll move the reverse reactions outside the TCA cycle.

[1] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8115731/

**Comment by deustp01** [comment]
> As to the reverse reactions, there is no UniProt annotation to "human GO:0019643". However, there are recent findings in [1] which would rather argue for a separate pathway.

Splitting is fine with me in principle. I need to study [1] some more (and cited references) to see if there is evidence that these additional reactions mediated by non-standard isoforms and running in non-standard directions have been shown to have a role in normal metabolism. If they have been observed only in disease processes, then (in the Reactome world) that would be an argument for a disease pathway counterpart of the existing normal TCA cycle pathway. In the GO-CAM world, at present, they might be out of scope.

I also remember reading discussion of the textbook TCA cycle that point out that most of the reactions of the cycle are reversible, and that in fact normal physiological conditions exist under which the reverse reaction might operate. I don't know anything about experimental data on this point. The standard textbook view also includes discussion of extensive traffic in and out of the TCA cycle at points other than the reaction of acetyl CoA with OAA to form citrate, to bring in carbon skeletons from amino acid metabolism to be catabolized, and perhaps to draw off carbon skeletons for biosyntheses (notably 2-oxoglutarate <=> glutamate?).

Perhaps to begin with, we should add all of the new reactions to a single super-TCA pathway in Reactome. Once we have all the pieces in place, it may be easier to decide whether they fit together as an augmented but still single pathway (I think that's the direction [1] is going in), or whether we can come up with a way to break the superpathway into several pieces, one of which will look a lot like textbook TCA cycle.

**Comment by ukemi** [comment]
For at least some of the enzymes, both directions are already included in Reactome. They made it difficult for me to make a causal flow in a GO-CAM though, so I stuck with the simple textbook verison (maybe dangerous). If we find they are physiologically important, I can take the Reactome super-pathway and split it into several GO-CAMs. I've done this in the past.

**Comment by deustp01** [comment]
> They made it difficult for me to make a causal flow in a GO-CAM though

I missed that. In the case of the reversible reactions in the path between glucose and pyruvate, we created both directions, but then chose the appropriate one of each to populate glycolysis and the other one to populate gluconeogenesis, and none were used in both pathways. Here, I still like the idea of building an enlarged TCA superpathway, then looking at it as a whole to try to work out some sensible subdivisions, but at the same time building many specific GO-CAMs both allows curation to proceed while we develop the overview, and will also be a really useful pilot project to identify plausible subdivisions.

**Comment by rwst** [comment]
Cytosolic reactions: IDH1 doesn't seem to be part of a hypothetical cytosolic TCA cycle because its product 2-OG would have to be reimported into the mitochondrial matrix. However, the only known transporter of 2-OG, SLC25A12, mainly exports it in exchange with oxoadipate under normal physiological conditions (Fiermonte et al., 2001).

This will be added to our IDH1 summation.

[1] https://pubmed.ncbi.nlm.nih.gov/11083877/

**Comment by rwst** [comment]
@deustp01 pointed me to this new paper:
Gnaiger, Erich. "Complex II ambiguities―FADH2 in the electron transfer system." Journal of Biological Chemistry (2023): 105470. https://www.sciencedirect.com/science/article/pii/S0021925823024985 (no PMID yet)

My overnight opinion:
Gnaiger is very much right to criticize illustrations containing succinate dehydrogenase (SDH). I agree with his Fig. 1e being the closest to what's happening. There are some loose ends which should be clarified, however. 

> H+-linked two-electron transfer from succinate to flavin adenine dinucleotide reduces the oxidized prosthetic group FAD to FADH2 with formation of fumarate. This H+-linked electron transfer through CII is not coupled to H+ translocation across the mtIM. Hence, CII is not a H+ pump
> SDHA catalyzes the oxidation succinate → fumarate + 2{H++e-} and reduction FAD+2{H++e-} → FADH2 in the soluble domain of CII.

So far so good. There is no mention, however, how the FADH2 moiety is recycled. On several occasions he says electrons are transferred through SDHB/the complex to reduce Q10 to Q10H2. But later this quote:

> In convergent electron transfer into the Q-junction, the independent part of CII played in the ETS is clarified by recognition of succinate (but not FADH2) as the substrate generated in the TCA cycle and feeding 2{H++e-} into the CII-branch of the Q-junction ([Figure 4](https://www.jbc.org/article/S0021-9258(23)02498-5/fulltext#gr4)b). 

Of course it would be convenient if the whole reduction equivalent 2{H++e-} were transferred from the FADH2 moiety to SDHC/SDHD/Q10 but is this the case? I recall reading always about the electron part visiting the three iron-sulfur clusters in between. Maybe the FADH2 moiety, while being on the cytosolic SDHA, is near enough the enzymatic center of membrane-localized SDHC/SDHD/Q10 that the proton transfer is one step, only the electrons have to divert through the clusters? If not, I wouldn't commit to talking about reduction equivalents but rather believe that the Fe-S clusters draw the electrons, at the same time freeing protons. And the Q10 reduction takes different protons from the environment.

Anyway, the paper serves well as reference for why most depictions of the SDH process shouldn't be followed blindly.

**Comment by deustp01** [comment]
Time to close, with issue of physiological roles of reversible TCA cycle reactions ([PMID: 36581208](https://pubmed.ncbi.nlm.nih.gov/36581208/)) to be revisited later
